examples/06-verif-manual/vm-007-plastic_compression_of_a_pipe_assembly.py

```diff
@@ -0,0 +1,578 @@
+r"""
+.. _ref_vm7_example:
+
+Plastic Compression of a Pipe Assembly
+--------------------------------------
+Problem Description:
+ - Two coaxial tubes, the inner one of 1020 CR steel and cross-sectional
+   area :math:`A_{\mathrm{s}}`, and the outer one of 2024-T4 aluminum alloy
+   and of area :math:`A_{\mathrm{a}}`, are compressed between heavy, flat end plates,
+   as shown below. Determine the load-deflection curve of the assembly
+   as it is compressed into the plastic region by an axial displacement.
+   Assume that the end plates are so stiff that both tubes are shortened by
+   exactly the same amount.
+
+Reference:
+ - S. H. Crandall, N. C. Dahl, An Introduction to the Mechanics of Solids,
+   McGraw-Hill Book Co., Inc., New York, NY, 1959, pg. 180, ex. 5.1.
+
```

Comment:
This is probably fine since this is the way we've been doing it for the other VMs. If we want to update it, let's do another follow-up PR for all the VMs.

Reply:
so i am going to leave it as in the previous VMs, and in the future we will fix it all in one fell swoop.

Reply:
Fair enough.